doc/man1/openssl.pod

```diff
@@ -75,7 +75,7 @@ B<list>, or B<no->I<XXX> itself.)
 
 =head2 Standard Commands
 
-=over 10
+=over 4
```

Comment:
At the end there are things like "cast cast-cbc" that are longer than 9. Anyway, I have no problem with the change, just making you aware of the effect this change has.